simulator/src/css/main.stylesheet.css

```diff
@@ -1368,6 +1368,7 @@ input:checked + .slider:before {
 
 .navbar .nav.pull-right {
     float: right;
+    margin-left: -106px;
```

Comment:
Not a good solution, it will be specifically for long names, we need to have a standard way for solving this.

So, I can see that in `.projectName` class we have defined width to be `35vw`, if we can remove that, it might solve, could you check this?

Reply:
I have checked and changes this thing could you please look into this again .